robottelo/common/constants.py

```diff
@@ -7,12 +7,17 @@
 
 FOREMAN_PROVIDERS = {
```

Comment:
@oshtaier May be we can add a note to show that we are using the sort list as per UI and not alphatetical.